File: clientJava/controllerApi/src/main/java/org/evomaster/clientJava/controllerApi/dto/AuthenticationDto.java
package org.evomaster.clientJava.controllerApi.dto;

import java.util.ArrayList;
import java.util.List;

/**
 * To authenticate a user, would need specific settings, like
 * specific values in the HTTP headers (eg, cookies)
 */
public class AuthenticationDto {

    /**
     * The name given to this authentication info.
     * Just needed for display/debugging reasons
     */
    public String name;

    /**
     * The headers needed for authentication
     */
    public List<HeaderDto> headers = new ArrayList<>();


    public AuthenticationDto() {
    }

    public AuthenticationDto(String name) {
        this.name = name;
    }
}

File: client-dotnet/src/EvoMaster.Controller.Api/dto/AuthenticationDto.cs
using System.Collections.Generic;

namespace EvoMaster.Controller.Api {
    /**
   * To authenticate a user, would need specific settings, like
   * specific values in the HTTP headers (eg, cookies)
   */
    public class AuthenticationDto {
        public AuthenticationDto() { }

        public AuthenticationDto(string name) {
            this.Name = name;
        }

        /**
     * The name given to this authentication info.
     * Just needed for display/debugging reasons
     */
        public string Name { get; set; }

        /**
     * The headers needed for authentication
     */
        public IList<HeaderDto> Headers { get; set; } = new List<HeaderDto>();

        /**
     * If the login is based on cookies, need to provide info on
     * how to get such a cookie
     */
        public CookieLoginDto CookieLogin { get; set; }
    }
}